docs/download.md

```diff
@@ -42,17 +42,17 @@ The plugin is tested on the following architectures:
 		Apache Spark 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.2.4
 		Apache Spark 3.3.0, 3.3.1, 3.3.2, 3.3.3
 		Apache Spark 3.4.0, 3.4.1
-		Apache Spark 3.5.0
+		Apache Spark 3.5.0, 3.5.1
 	
 	Supported Databricks runtime versions for Azure and AWS:
-		Databricks 10.4 ML LTS (GPU, Scala 2.12, Spark 3.2.1)
 		Databricks 11.3 ML LTS (GPU, Scala 2.12, Spark 3.3.0)
 		Databricks 12.2 ML LTS (GPU, Scala 2.12, Spark 3.3.2)
 		Databricks 13.3 ML LTS (GPU, Scala 2.12, Spark 3.4.1)
 	
-	Supported Dataproc versions:
+	Supported Dataproc versions (Debian/Ubuntu):
 		GCP Dataproc 2.0
 		GCP Dataproc 2.1
+		GCP Dataproc 2.2
```

Comment:
[Dataproc 2.2](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.2) uses Spark 3.5.0 (Debian 12 and Ubuntu 22). I've updated our init scripts to support NVIDIA drivers installation on latest Debian, Ubuntu versions. I ran churn example but didn't run the integration tests. Please let me know if we can add it in the list of supported runtimes.

Reply:
Since we are not doing integration tests on Debian, I would not say that it is supported.

Reply:
Noted, thanks!